wiki/History_of_osu!/1/en.md

```diff
@@ -128,12 +148,12 @@ Continuing the trend set by July, one would be hard pressed to find an aspect of
 
 **Personal online best scores were added to song selection**, fulfilling a long-time request. This required some fairly hefty database optimisation. Graphical improvements were made to the ranking screen, and various elements of the default skin, in an aim to make it feel more clean and crisp than before. The display of **slider tick score sprites (10/30) was implemented.** Improvements were seen in the multiplayer match setup, allowing **host control to be transferred**. Many **new chat commands** were added for user convenience.
 
-A new ranked status "![](fire.gif "fig:fire.gif")Approved" was added for maps which are of great quality but exceed ranking length or difficulty (or other) limits in some way. Maps in this status don't add to player's scores, but have scoreboards and act otherwise like a ![](heart.gif "fig:heart.gif")ranked map.
+A new ranked status "![Fire.gif](img/fire.gif "Fire.gif")Approved" was added for maps which are of great quality but exceed ranking length or difficulty (or other) limits in some way. Maps in this status don't add to player's scores, but have scoreboards and act otherwise like a ![Heart.gif](img/heart.gif "Heart.gif")ranked map.
```

Comment:
~~broken images~~
these images are in the shared folder; replace entire line with
```
A new ranked status ![Fire](/wiki/shared/Fire.gif "Fire") **approved** was added for maps which are of great quality but exceed ranking length or difficulty (or other) limits in some way. Maps in this status don't add to player's scores, but have scoreboards and act otherwise like a ![heart](/wiki/shared/Heart.gif "heart") **ranked** map.
```